### ISSUE TITLE
Old method removed?

### ISSUE THREAD

**Issue by Barry-Bot1** [TheLastBen/fast-stable-diffusion#396]
Has the old method been removed? I found it very useful for training styles :(

**Comment by TheLastBen** [comment]
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/Dreambooth/fast_DreamBooth-Old-Method.ipynb

**Comment by Gyro0o** [comment]
I agree the old method is more predictable in styles

**Comment by TheLastBen** [comment]
if you rename the instance pictures correctly, you can easily mimic the old method with the new method

**Comment by LIQUIDMIND111** [comment]
> https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/Dreambooth/fast_DreamBooth-Old-Method.ipynb

so the OLD method was completely removed?????

**Comment by TheLastBen** [comment]
Yes, no longer maintained, the new method is way better and easier, keep the total steps of the text encoder below 1000 and you will have great results.

**Comment by LIQUIDMIND111** [comment]
> Yes, no longer maintained, the new method is way better and easier, keep the total steps of the text encoder below 1000 and you will have great results.

oh ok, i thought FOR PERSON it needed 70 % - "Train_text_encoder_for:
70
If you're training a style, keep it between 10-20%, if you're training on a person, set it between 50-70%, reduce it if you can't stylize the person/object.
Higher % will give more weight to the instance, it gives stronger results at lower steps count, but harder to stylize."

**Comment by TheLastBen** [comment]
keep it 15% and see the results
